### PR TITLE
Update cookbooks to enable use with Ubuntu 14.04LTS & Drupal8.

### PR DESCRIPTION
**Pull request by arknoll**
https://github.com/vampd/drupal/issues/24

### CODE REVIEW

File: recipes/mysql.rb [hunk 1]
```diff
@@ -19,42 +19,39 @@
...
+
+mysql2_chef_gem 'default' do
+  action :install
 end
 
 mysql_connection_info = {
-  :host => node[:db][:host],
-  :username => node[:db][:root],
-  :password => node[:mysql][:server_root_password]
+  :host => '127.0.0.1',
```
Comment (timodwhit) [inline recipes/mysql.rb]: Do we use the default attributes for host/port/etc?

File: recipes/php.rb [hunk 2]
```diff
@@ -17,18 +17,26 @@
...
+  action :create
+  notifies :install, "php_pear[apc]", :immediately
 end
 
-# install apc pecl with directives.
-php_pear "apc" do
-  action :install
+# Install apc pecl with directives.
+php_pear 'apc' do
+  action :nothing
```
Comment (timodwhit) [inline recipes/php.rb]: Should this be install if php = 5.3, instead of nothing?

File: metadata.rb [hunk 3]
```diff
@@ -4,13 +4,14 @@
 license          'Apache 2.0'
 description      'Manages Drupal'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.0.7'
+version          '5.0.0'
```
Comment (timodwhit) [inline metadata.rb]: Is this really a non backwards compatiable change?

File: recipes/mysql.rb (hunk 1 above)
Comment (timodwhit) [inline recipes/mysql.rb]: Specifically should we utilize or have our own settings for this?
```
default[:drupal][:sites][site_name][:drupal][:settings][:db_host] = '127.0.0.1'
default[:drupal][:sites][site_name][:drupal][:settings][:db_port] = '3306'
```

Comment (arknoll) [inline recipes/mysql.rb]: There are no defaults anymore. It is all LWRPerized. These vars are neccessary.

File: recipes/php.rb (hunk 2 above)
Comment (arknoll) [inline recipes/php.rb]: This above: notifies :install, "php_pear[apc]", :immediately
takes care of that.

It ensures that php_pear 'apc' will only run after the ruby block. And then only if the ruby_block above sets :drupal:php:version to 5.3

Comment (timodwhit) [inline recipes/php.rb]: Ah, that makes more sense. My bad, didn't see that

File: metadata.rb (hunk 3 above)
Comment (arknoll) [inline metadata.rb]: There are a lot of changes with this one. I think it would introduce too many headaches to try and say it is backwards compatible.

Comment (timodwhit) [inline metadata.rb]: But the question is: Are there any changes to the api? 4.1 might be good release since there are no big breaking changes, right?

Comment (arknoll) [inline metadata.rb]: You can't run the old vampd with the new drupal.

You can't run the new vampd with the old drupal.

You can run the new vampd with the new drupal.
You can run the old vampd with the old drupal.

**Pull request merged**